test/unit/connect-inject-clusterrole.bats

```diff
@@ -74,110 +74,15 @@ load _helpers
 }
 
 #--------------------------------------------------------------------
-# global.acls.manageSystemACLs for namespaces
```

Comment:
Does it make sense to add a test to cover the new clusterrole rules, e.g. the addition of the `nodes` resource?
 `resources: ["pods", "nodes"]`

Reply:
Typically we don't write tests for non-conditionals, e.g. we would write a test for:
```
- resources: ["pods"{{ if .something}}, "nodes"{{end}}]
```

to check that nodes isn't added unless `.something`.

But we wouldn't write a test for:
```
- resources: ["pods", "nodes"]
```

Because at that point you're testing that the template is as you wrote it, which would lead to having to test every line in the template which is too much testing.

For example here we have all these resources: https://github.com/hashicorp/consul-helm/blob/master/templates/controller-clusterrole.yaml#L12-L23 but because they're not templated in a conditional, we don't test that they're set in https://github.com/hashicorp/consul-helm/blob/master/test/unit/controller-clusterrole.bats.

Reply:
cool, makes sense! no worries then :)